src/Conference/ConferenceDetails.jsx

```diff
@@ -6,10 +6,39 @@ import { Link, useParams } from "react-router-dom";
 const StyledConferenceDetails = styled.div`
   width: 100%;
   height: 400px;
-  color: white;
-  background: darkslategray;
+
+`;
+
+const StyledDetailsCardHeading = styled.h3`
+background: #dbd8db;
+width: 100%;
+height: 55px;
+font-size: 30px;
+font-weight: bold;
+text-align: center;
+vertical-align: middle;
+
+
 `;
 
+const StyledDetailsCard = styled.div`
+  border-style: solid;
+  border-color: black;
+  border-radius: 25px;
+  background: ${props => props.background || "#dbd8db"};
```

Comment:
I'm not sure you need props here as you're not passing any props to the component!